src/btree.c

```diff
@@ -3083,13 +3083,56 @@ btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_btree_pivot_stats(platform_log_handle *log_handle,
+                              btree_pivot_stats   *pivot_stats)
+{
+   // If nothing has been updated, yet, print a msg accordingly.
+   if ((pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER))
+   {
+      platform_log(log_handle, "   (Stats=BTREE_UNKNOWN_COUNTER)\n");
```

Comment:
While exercising different print methods, I found that many times these BTree stats are in this "uninitialized state" till the time BTree-Pack is done. So, I added this check to capture this case and [to] report L3095.

(Btw, this change sets up the stage that the changes under PR #512 will bring-in to print the `key_bytes` and `message_bytes` as values with unit-specifiers. If I did that for this uninitialized state, we will get a very large "(2 GiB)" reported in the outputs, which will be misleading.)

Reply:
The output when exercising these print methods thru the 

`VERBOSE=1 build/release/bin/unit/splinter_test test_splinter_print_diags` 

test is as follows:

```
    23 Memtable root_addr=48: gen 5615779840 ref_count 3 state 1
    24 *******************
    25 BTree node at addr=5615779840
    26 {
    27 **  INDEX NODE
    28 **  Header ptr: 0x7fc56e5a9000
    29 **  addr: 5615779840
[...]

    40 Array of 4 index entries:
    41 [ 0]: key=0x00001c2c38bd531100000000000000000000000000000000
    42    child_addr=5616173056
    43    (Stats=BTREE_UNKNOWN_COUNTER)
...
```

This test case simply does a bunch of inserts, and BTree-pack is not explicitly being invoked. Hence, these pivot-stats values are still in their uninitialized state.